psi4/src/psi4/libmints/extern.cc

```diff
@@ -92,6 +92,25 @@ void ExternalPotential::print(std::string out) const {
     }
 }
 
+std::string ExternalPotential::get_xyz() {
```

Comment:
Please rename this to signify that it only gives coordinates for point charges.

Reply:
Fixed.